src/CatalogService.Core/Models/CategoryModel.cs

```diff
@@ -0,0 +1,3 @@
+namespace CatalogService.Core.Models;
+
+public record CategoryModel(string Name, string? Image, string? Parent);
```

Comment:
As it is now it would simply ignore as I didn't create a reference for that, I'll update the script in order to create that dependency, thus if you pass a value for Parent that isn't consistent with the Categories registered on the table it will throw an error